distributed/comm/tcp.py

```diff
@@ -124,7 +124,10 @@ def _get_finalizer(self):
         def finalize(stream=self.stream, r=repr(self)):
             if not stream.closed():
                 logger.warn("Closing dangling stream in %s" % (r,))
-                stream.close()
+                try:
+                    stream.close()
+                except socket.error:
```

Comment:
Which error specifically did you get here? It would be nice at least to comment about it.